refactor(share_plus)!: Share API cleanup (#2832)
fluttercommunity/plus_plugins: packages/share_plus/share_plus/README.md

To use this plugin, add `share_plus` as a [dependency in your pubspec.yaml file](https://plus.fluttercommunity.dev/docs/overview).

Import the library.

```dart
import 'package:share_plus/share_plus.dart';
```

### Share Text

> Invoke the static `share` method anywhere in your Dart code.

Reviewer: ```suggestion
Invoke the static `share()` method anywhere in your Dart code.
```
What about adding braces so it a more obvious that it is a method?

Share.share('check out my website https://example.com', subject: 'Look what I made!');
```

> If you are interested in the action your user performed with the share sheet, the `share` method returns a status object.

Reviewer: ```suggestion
`share()` returns `status` object that allows to check the result of user action in the share sheet.
```

Not sure if it would be better, but felt to me so.
